problem.py

```diff
@@ -133,23 +160,33 @@ def _read_data(path, train_or_test="train"):
 
 
 def _check_step_list(step_list):
-    """Some sanity checks."""
+    """Some sanity checks. (step_list is assumed to be sorted."""
```

Comment:
Extra parenthesis.

Reply:
To make sure you have a sorted list, you can sort the prediction no?

`sorted([tuple(step) for step in step_list_pred])` will get you a list sorted with lexicographic order to make sure you don't fail because it is not properly sorted.

Reply:
you are right, no need to have them sorted. Correcting right now.

Reply:
solved in 94aab1998d5e224b10665327e12b17e62fa4d644